### ISSUE TITLE
Add item drop quantity/frequency static results

### ISSUE THREAD

**Issue by Violin9906** [penguin-statistics/frontend-v2#208]
### What problem does this feature solve?
With this feature, I can use Markov Dynamic Programming to find the best choice when single type of item is needed. The static result of item drop quantity/frequency instead of drop precentage is indispensible for a stochastic matrix.

### What does the proposed API look like?
Give the static results of item drop quantity/frequency like this:
```javascript
{
  stage: "1-7",
  items: {
      "Orirock Cube": [
          {quantity:1, frequency: 0.8},
          {quantity:2, frequency: 0.2} // just for a example
      ],
      "Orirock": ......
    }
}
```

**Comment by GalvinGao** [comment]
Will be implemented through the *Statistics of a Stage* feature. By then such feature will include the drop combinations, along with their hit counts and percentages.

**Comment by GalvinGao** [comment]
Has been implemented in version *candidate* v3.3.7. Such version would be release soon ;)

Thanks for your proposal! `:D`
